client/ayon_core/plugins/publish/extract_thumbnail.py

```diff
@@ -360,6 +361,31 @@ def _get_filtered_repres(self, instance):
             filtered_repres.append(repre)
         return filtered_repres
 
+    def _is_valid_images_repre(self, repre):
+        """Check if representation contains valid image files
+
+        Args:
+            repre (dict): representation
+
+        Returns:
+            bool: whether the representation has the valid image content
+        """
+        files = repre.get("files")
+        if not files:
+            self.log.debug((
+                "Representation \"{}\" doesn't have files. Skipping"
+            ).format(repre["name"]))
+            return False
+
+        # Get first file's extension
+        if isinstance(files, (list, tuple)):
+            first_file = files[0]
+        else:
+            first_file = files
+
+        ext = os.path.splitext(first_file)[1].lower()
+        return ext in IMAGE_EXTENSIONS
```

Comment:
Requires import of `VIDEO_EXTENSIONS`
```suggestion
        return ext in IMAGE_EXTENSIONS or ext in VIDEO_EXTENSIONS
```